src/main/java/org/springframework/integration/kafka/support/KafkaHeaders.java

```diff
@@ -28,8 +29,12 @@
 
 	public static final String MESSAGE_KEY = PREFIX + "messageKey";
 
-	public static final String PARTITION_ID = PREFIX + "_partitionId";
+	public static final String PARTITION_ID = PREFIX + "partitionId";
 
-	public static final String OFFSET = PREFIX + "_offset";
+	public static final String OFFSET = PREFIX + "offset";
```

Comment:
Probably the best idea is to segregate that fix and have manual ack supported in SI-K 1.0 and SI K 1.1 (since XD 1.1 will seemingly need manual acks and we don't want to force a SI-K upgrade until 1.2)